app/models/event_blacklisting.rb

```diff
@@ -0,0 +1,76 @@
+class EventBlacklisting < ActiveRecord::Base
```

Comment:
There was probably a discussion I missed on the naming of `EventBlacklisting`.  Looking up the meaning of a `listing` shows `a printed list` as the main definition.  To avoid ambiguity I would change this name to `EventBlacklistEntry`